promenade/templates/genesis/etc/kubernetes/armada-loader/pki/armada-loader-key.pem

```diff
@@ -0,0 +1 @@
+{{ config.get(kind='CertificateKey', name='admin')['data'] }}
```

Comment:
These are given as inputs to genesis and join scripts.  They are currently generated by the `promenade generate` command, but the long-term plan for their source is probably that will be provided externally.